src/components/DetailsPage.jsx

```diff
@@ -20,7 +20,7 @@ function DetailsPage() {
           (countrie) => (
             <div key={nanoid()} className="country-card">
               <div className="home-container">
-                <NavLink to="/"><img src={home} alt="home" className="home" /></NavLink>
+                <NavLink to="/countries-database/"><img src={home} alt="home" className="home" /></NavLink>
```

Comment:
- [ ] Great job implementing the required changes I requested you thank you so much but there are still some issues you need to address before this project can be approved . According to the requirement, It's possible to navigate the detail page by clicking the individual category item, there's a link from the Detail page to the Home page by clicking the "Back" button. please fix this
 given desine

- [ ] 
<img width="421" alt="metrics_app" src="https://github.com/CarlosZ96/countries-database/assets/92458236/2a76c4a7-ff66-4866-a0b3-9315d2d5b10f">

- [ ] yur design screenrec
![chrome-capture-2023-8-24](https://github.com/CarlosZ96/countries-database/assets/92458236/5844ed02-334a-4b59-ad45-c79aa03e029e)